### PR TITLE
Merge 20.1 into 20.1.1

### PR DESCRIPTION
**Pull request by malinajirka**
This PR merges 20.1 into 20.1.1.

The 20.1 branch contains a fix for a critical issue that was merged after 20.1 was build. We decided to stop the release process and proceed with a hotfix release scenario. The version 20.1 was never actually shipped to users.

This PR contains the fix for the critical issue. However, it also contains changes to translations which I'm not 100% sure are expected - wdyt @AliSoftware ?

P.S. The CI complains about a version code but that is expected - it is fixed in https://github.com/woocommerce/woocommerce-android/pull/12473 which hans't been merged yet.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR has the <code>Releases</code> label: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by AliSoftware** [comment]
That is a good point, we should probably revert the changes (and especially the removals) made on translations.

I think what happened here is that when you tried to re-run `finalize_release` on `20.1` earlier today (before we decided to go with the hotfix route after all), it also re-downloaded the latest translations before it tried to do the new `20.1` build… but because you already ran the code freeze for `20.2` — which sends the new `values/strings.xml` to GlotPress during code freeze — then deletions of some obsolete strings that happened on `trunk` during the sprint where you were working on `20.2` were thus deleted from GlotPress… and thus their translations were also removed when the `finalize_release` from `20.1` tried to download the latest translations…?

TL;DR: I think we'd be safer to discard / revert the commit that uploaded the translations from this PR indeed.

**Comment by malinajirka** [comment]
> TL;DR: I think we'd be safer to discard / revert the commit that uploaded the translations from this PR indeed.

@AliSoftware  I'm wondering if a revert commit causes any issues down the line => we plan to merge 20.1.1 into 20.2 and eventually into trunk => won't the revert commit be propagated during the process? I believe it will, so perhaps what might be safest is cherry pick the fix for the critical issue onto 20.1.1, wdyt?


Update: I opted for merging only the bug fix into 20.1.1 instead of merging all the changes. Closing this PR in favor of https://github.com/woocommerce/woocommerce-android/pull/12475.

**Comment by malinajirka** [comment]
Closed in favor of https://github.com/woocommerce/woocommerce-android/pull/12475
